.github/workflows/docs.yml

```diff
@@ -0,0 +1,57 @@
+name: Docs
+on:
+  pull_request:
+    types: [synchronize, edited, opened]
+  push:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Poetry
+        run: pipx install poetry
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          cache: poetry
```

Comment:
We can also set it to `3.x` and it would just use the latest. I'm not TOO worried about a Python 4.0 happening anytime soon *hears the gnashing of teeth of python developers in the year 2008*

Reply:
I like 3.x! :+1: (I didn't know that was an option...)